app/helpers/file_icons_helper.rb

```diff
@@ -22,6 +22,10 @@ def file_fa_icon_class(asset)
       'fa-file-pdf'
     elsif %w(txt csv tab tex).include?(file_ext)
       'far fa-file-alt'
+    elsif asset.file.attached? && asset.file.metadata['asset_type'] == 'marvinjs'
+      'mce-i-marvinjs'
+    elsif asset.file.attached? && asset.file.metadata['asset_type'] == 'gene_sequence'
```

Comment:
We do not need this. This method is for font awesome icons. For gene sequence we use a custom icon